src/models/invites-ignorer-types.ts

```diff
@@ -29,6 +29,19 @@ export const IGNORE_INVITES_ACCOUNT_EVENT_KEY = new UnstableValue(
     "org.matrix.msc3847.ignore.invites",
 );
 
+export interface Policies {
+    target?: string;
+    sources?: string[];
+}
+
+export interface IgnoreInvitesContent {
+    "m.ignore.invites": Policies;
+}
+
+export interface UnstableIgnoreInvitesContent {
+    "org.matrix.msc3847.ignore.invites": Policies;
+}
```

Comment:
again, please can we make a habit of writing documentation for new types that we introduce, *especially* when they are `export`ed outside the file they are defined in? These are the content of particular types of account data, correct?